Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM alpine:3.5
+
+ADD ./pv-cleaner-operator /pv-cleaner-operator
```

Comment:
You could add also a `.dockerignore` file to make the build context smaller and increase build performance, see https://github.com/giantswarm/aws-operator/blob/master/.dockerignore